src/views/OrderDetail.vue

```diff
@@ -4,178 +4,239 @@
       <ion-toolbar>
         <ion-back-button default-href="/" slot="start" />
         <ion-title>{{ $t("Order details") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button :disabled="order?.rejected" @click="openOrderItemRejHistoryModal()">
+            <ion-icon slot="icon-only" :icon="timeOutline" />
+          </ion-button>
+          <ion-button class="ion-hide-md-up" :disabled="!hasPermission(Actions.APP_ORDER_UPDATE) || order?.readyToHandover || order?.rejected" @click="updateOrder(order)">
+            <ion-icon slot="icon-only" color="danger" :icon="bagRemoveOutline" />
+          </ion-button>
+        </ion-buttons>
       </ion-toolbar>
     </ion-header>
+
     <ion-content>
       <main>
-        <ion-list>
-          <ion-item lines="none">
+        <aside>
+          <ion-item v-if="order?.readyToHandover || order?.rejected" color="light" lines="none">
+            <ion-icon :icon="order?.readyToHandover ? checkmarkCircleOutline : order?.rejected ? closeCircleOutline : ''" :color="order?.readyToHandover ? 'success' : order?.rejected ? 'danger' : ''" slot="start" />
```

Comment:
There is no need to have the extra check for order.rejected in icon ternary condition as this block will only be displayed when we have readyToHandover or rejected order.
Also, conditional chaining is also not required inside the ion-item.